config.yaml

```diff
@@ -108,6 +108,7 @@ matrix:
           - css
           - cts_skip_unver
           - cts
+          - dataarts
```

Comment:
@mmazec you want to enable this metric to be monitored only from eu-de to eu-de
or also from eu-nl to eu-de?

Reply:
@vladimirhasko added monitoring zone from EU-NL.